src/tracer.js

```diff
@@ -95,10 +100,30 @@ export default class Tracer {
 
         span.addTags(tags);
         span.addTags(internalTags);
+
+        // emit metrics
+        this._metrics.spansStarted.increment(1);
+        if (span.context().isSampled()) {
+            this._metrics.spansSampled.increment(1);
+            if (!hadParent) {
+                this._metrics.tracesStartedSampled.increment(1);
+            } else if (rpcServer) {
+                this._metrics.tracesJoinedSampled.increment(1);
+            }
+        } else {
+            this._metrics.spansNotSampled.increment(1);
+            if (!hadParent) {
```

Comment:
you can add a check for emitted metrics in the unit test where you test jaeger-debug-id header - the span created in that test should emit "trace started" metrics, but I think your code will emit "trace joined".